db/migrate/20250504195512_drop_rubric_gradings.rb

```
@@ -0,0 +1,4 @@
+class DropRubricGradings < ActiveRecord::Migration[6.1]
+  def change
+  end
```

RuboCop (1.73)

[convention] Put empty method definitions on a single line.

(Style/EmptyMethod)